test/e2e/features/minimal.feature

```diff
@@ -10,9 +10,22 @@ Feature: Minimal user story
         And starting CRC with default bundle succeeds
         Then checking that CRC is running
 
+        @podman-preset
         Examples:
             | preset-value |
             | podman       |
+
+        @microshift-preset
+        Examples:
+            | preset-value |
             | microshift   |
+
+        @openshift-preset
+        Examples:
+            | preset-value |
             | openshift    |
 
+
```

Comment:
why all these empty lines?

Reply:
future additions ;-)

Reply:
Easier to read the file I guess. Otherwise it's all one block. `Example` blocks are always separated from above by an empty line, so it seemed sensible to separate them like that from each other too.

Reply:
We referred to the additional bottom 4 lines.